src/calculateRentalCost.js

```diff
@@ -26,9 +26,19 @@
  *
  * @return {number}
  */
-
+ 
 function calculateRentalCost(days) {
-  // write code here
+  // This function calculates a discount on car rental depending on the number of rental days
+  let totalCost = days * 40;
+
+  const discountCost
+      = days > 2 && days < 7
```

Comment:
Instead of using nested ternary operators, consider using if-else statements to improve readability.